packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_transmission.ts

```diff
@@ -0,0 +1,91 @@
+
+const NAME = "KHR_materials_transmission";
+
+/**
+ * [Specification](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_transmission/README.md)
+ */
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export class KHR_materials_transmission implements IGLTFExporterExtensionV2 {
+    /** Name of this extension */
+    public readonly name = NAME;
+
+    /** Defines whether this extension is enabled */
+    public enabled = true;
+
+    /** Defines whether this extension is required */
+    public required = false;
+
+    private _exporter: _Exporter;
+
+    private _wasUsed = false;
+
+    constructor(exporter: _Exporter) {
+        this._exporter = exporter;
+    }
+
+    public dispose() {}
+
+    /** @hidden */
+    public get wasUsed() {
+        return this._wasUsed;
+    }
+
+    public postExportMaterialAdditionalTextures?(context: string, node: IMaterial, babylonMaterial: Material): BaseTexture[] {
+        const additionalTextures: BaseTexture[] = [];
+
+        if (babylonMaterial instanceof PBRMaterial) {
+            if (this._isExtensionEnabled(babylonMaterial)) {
+                if (babylonMaterial.subSurface.thicknessTexture) {
+                    additionalTextures.push(babylonMaterial.subSurface.thicknessTexture);
+                }
+                return additionalTextures;
+            }
+        }
+
+        return additionalTextures;
+    }
+
+    private _isExtensionEnabled(mat: PBRMaterial): boolean {
+        // This extension must not be used on a material that also uses KHR_materials_unlit
+        if (mat.unlit) {
+            return false;
+        }
+        const subs = mat.subSurface;
+        return (subs.refractionIntensity != undefined && subs.refractionIntensity != 0) || this._hasTexturesExtension(mat);
+    }
+
+    private _hasTexturesExtension(mat: PBRMaterial): boolean {
+        return mat.subSurface.refractionIntensityTexture != null;
+    }
+
+    public postExportMaterialAsync?(context: string, node: IMaterial, babylonMaterial: Material): Promise<IMaterial> {
+        return new Promise((resolve) => {
+            if (babylonMaterial instanceof PBRMaterial && this._isExtensionEnabled(babylonMaterial)) {
+                this._wasUsed = true;
+
+                const subs = babylonMaterial.subSurface;
+                const transmissionFactor = subs.refractionIntensity == 0 ? undefined : subs.refractionIntensity;
+                const transmissionTexture = this._exporter._glTFMaterialExporter._getTextureInfo(subs.refractionIntensityTexture) ?? undefined;
+
+                const volumeInfo: IKHRMaterialsTransmission = {
+                    transmissionFactor: transmissionFactor,
+                    transmissionTexture: transmissionTexture,
+                    hasTextures: () => {
+                        return this._hasTexturesExtension(babylonMaterial);
+                    },

```

Comment:
Ugh, I didn't realize this was added to the gltf2interfaces. This shouldn't be there. It's an existing problem, so we can address later.

Reply:
Well, i just duplicate and implements the interface
```
    /** @hidden */
    interface IMaterialExtension {
        hasTextures?(): boolean;
    } 
```

Reply:
Yes, I see that. This is an implementation detail of serializer and shouldn't be in the interfaces.